examples/function-pointers/function_pointers.bal

```diff
@@ -2,35 +2,39 @@ import ballerina/io;
 import ballerina/lang.'int as ints;
 
 // The `test` function acts as a variable function pointer in the `main` function.
-function test(int x, string s) returns float {
+function test(string s, int... x) returns float {
     int | error y = ints:fromString(s);
     float f = 0.0;
 
     if (y is int) {
-        f = x * 1.0 * y;
+        foreach var item in x {
+            f += item * 1.0 * y;
+        }
     } else {
         // The type of `y` within the else block would be `error`.
         panic y;
     }
     return f;
 }
 
-// A function pointer as a parameter. Uses the `.call()` method to invoke the function using the function pointer.
-function foo(int x, function (int, string) returns float bar)
+// A function pointer as a parameter. A function pointer can be invoked similar to how a normal function is invoked.
+function foo(int x, function (string, int...) returns float bar)
              returns float {
-    return x * bar(10, "2");
+    return x * bar("2", 2, 3, 4, 5);
 }
 
 // A function pointer as a return type.
-function getFunctionPointer() returns (function (int, string) returns float) {
+function getFunctionPointer() returns (function (string, int...) returns float) {
```

Comment:
These lines are greater than 80 character width, which will make it not clear with random line breaks at b.io site. So limit the line width to max of 80 chars. Refer for more details : https://github.com/ballerina-platform/ballerina-lang/blob/master/docs/bbe-guidelines/bbe-guidelines.md

Reply:
Resolved.